lib/sidekiq/statistic/statistic/metrics/store.rb

```diff
@@ -19,8 +19,7 @@ def call
           cache_length = 0
 
           Sidekiq.redis do |redis|
-            redis.pipelined { cache_length = store_cache_metrics(redis) }
-
+            redis.pipelined { |pipeline| cache_length = store_cache_metrics(pipeline) }
```

Comment:
Did you have a chance to check if this solution works for the previous versions? We can use `Sidekid::VERSION` along with `Gem::Version` to check that.